tests/phpunit/tests/theme.php

```diff
@@ -1058,6 +1058,102 @@ static function () {
 		);
 	}
 
+	/**
+	 * Test whether a switched site retrieves the correct stylesheet directory.
```

Comment:
```suggestion
	 * Tests whether a switched site retrieves the correct stylesheet directory.
```
Minor nitpick: Docblocos for methods (tests too) should use third-person singular verbs. [Reference](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#language)

:1234: Applies to the other test methods.

Reply:
Thanks @costdev! I pushed this up in a4ed497